packages/airnode-abi/src/decoding.test.ts

```diff
@@ -102,4 +102,10 @@ describe('decode', () => {
   it('throws an error when invalid data is provided', () => {
     expect(() => decoding.decode('0x123456')).toThrowError();
   });
+
```

Comment:
I would maybe also add a test for the example in Buraks comment

Reply:
> I would maybe also add a test for the example in Buraks comment

Added these